common/src/main/scala/error/Errors.scala

```diff
@@ -5,5 +5,5 @@ case class UnsupportedPlatform(platform: String) extends RequestError {
 }
 
 case class MalformattedRegistration(description: String) extends RequestError {
-  override def reason: String = s"Malformatred request: $reason"
+  override def reason: String = s"Malformatred request: $description"
```

Comment:
@alexduf good thing you suggested the curl test .. 
(presumably the default case here: https://github.com/guardian/mobile-n10n/blob/master/registration/app/registration/services/LegacyRegistrationConverter.scala#L15 has never been reached ..